src/manifest.loader.ts

```diff
@@ -1,16 +1,50 @@
 import { fsa } from '@linzjs/s3fs';
-import { promises as fs } from 'fs';
+import * as fs from 'fs';
+import { join } from 'path';
 import { logger } from './log';
 import { Manifest, ManifestFile } from './manifest';
 
 const BackupExtension = '.1';
+export const MANIFEST_FILE_NAME = 'manifest.json';
 
 function isManifestPath(fileName: string): boolean {
   if (fileName.endsWith('.json' + BackupExtension)) return true;
   if (fileName.endsWith('.json')) return true;
   return false;
 }
 
+/**
+ * Return true if manifest dataPath are different
+ *
+ * @param manifestA
+ * @param manifestB
+ * @returns
+ */
+export function isManifestsDifferent(manifestA: ManifestLoader, manifestB: ManifestLoader): boolean {
+  if (manifestA.dataPath !== manifestB.dataPath) return true;
```

Comment:
We need to validate the contents of the manifests, primarilly.

- All files that are in the mainfestA must be in manifestB
- All hashes must match if they exist
- All file sizes must match

Possible:
- Additional files can exist in one of the manifests, but only one direction so a manifest in s3  vs a local manifest t he local manifest can contain additional files